dev/devicelab/lib/tasks/run_tests.dart

```diff
@@ -11,21 +11,104 @@ import '../framework/framework.dart';
 import '../framework/task_result.dart';
 import '../framework/utils.dart';
 
+TaskFunction createAndroidRunReleaseTest() {
+  return AndroidRunOutputTest(release: true);
+}
+
 TaskFunction createMacOSRunReleaseTest() {
   return DesktopRunOutputTest(
     // TODO(cbracken): https://github.com/flutter/flutter/issues/87508#issuecomment-1043753201
     // Switch to dev/integration_tests/ui once we have CocoaPods working on M1 Macs.
     '${flutterDirectory.path}/examples/hello_world',
     'lib/main.dart',
     release: true,
+    allowStderr: true,
```

Comment:
The macOS `flutter run` test has stderr output from xcode build. The original test from before this refactoring also [ignored stderr output](https://github.com/flutter/flutter/blob/7020f59ace273d8644f0c8f3d0c6e7c9c95ba51a/dev/devicelab/bin/tasks/run_release_test_macos.dart#L74).

Reply:
@stuartmorgan @cbracken do you remember which issue is tracking cleaning up the macOS Xcode -> stderr mapping?  I can't find it at the moment...

Reply:
I don't remember if we actually filed anything for that specifically.